### ISSUE TITLE
Update-DbaInstance is not picking up the latest SQL2014 and SQL2016 securityfix

### ISSUE THREAD

**Issue by elosdb** [sqlcollaborative/dbatools#6373]
We put the KB4535706 executable into the directory D:\Temp\SP2_CU11
Even after updating DbaTools the statement underneath is not picking up 4535706 Security update for SQL Server 2016 SP2 CU11: February 11, 2020  

Update-DbaInstance -ComputerName <servername> -Version SP2CU11 -Authentication 'Kerberos' -Path 'D:\Temp\SP2_CU11' -Restart -Confirm:$false -ExtractPath 'D:\temp' 

Results in a  warning: WARNING: [08:56:17][Update-DbaInstance] Could not find installer for the SQL2016 update KB4527378

**Comment by nvarscar** [comment]
Has the file pattern changed? The function looks for specific filename pattern. If the file doesn't match it, it won't be discovered.

**Comment by koglerk** [comment]
Try manually specifying the `-KB` parameter instead of using `-Version SP2CU11`. Does that change your result?

I happened to have both a 2016 and a 2014 server in my dev environment that needed to be patched today, so I tried applying the Feb 11 patch with `Update-DbaInstance` and it worked just fine.

Here's the 2016 server:

![image](https://user-images.githubusercontent.com/25352226/75073002-3355c580-54be-11ea-84bf-55c882db3674.png)

And the 2014 (with better formatting!):

![image](https://user-images.githubusercontent.com/25352226/75074960-61d59f80-54c2-11ea-97bc-10f2b6d94ba3.png)

Screenshots above running dbatools version 1.0.97
